pkg/printers/json.go

```diff
@@ -52,6 +52,10 @@ func (p *JSONPrinter) PrintObj(obj runtime.Object, w io.Writer) error {
 		return err
 	}
 
+	if obj.GetObjectKind().GroupVersionKind().Empty() {
```

Comment:
@juanvallejo do we have tests for these somewhere here, I can't find anything except for those in ` pkg/kubectl/cmd/cmd_printing_test.go`?

Reply:
@soltysh 
> do we have tests for these somewhere here

I believe these are the only ones I've come across so far: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/create/create_role_test.go#L108